distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -91,19 +91,24 @@ to:
 RUN_AS_USER=<${ddf-branding-lowercase}-user>
 ----
 +
-. Set the memory in the wrapper config to match with ${branding} default memory setting.
-.. Add the setting for PermGen space under the JVM Parameters section.
-.. Update heap space to 2048.
```

Comment:
Thanks @charlesmoore99 yes, I see where we keep that for the wrappers max memory.